frontend/src/Interface/LeafTestInterface.tsx

```diff
@@ -0,0 +1,13 @@
+/**
+ * @summary Interface for the main data file
+ * @description This interface defines how the main data file should be structured,
```

Comment:
```suggestion
 * @summary     Interface for the main data file
 * @description This interface defines how the main data file should be structured,
```

You don't actually have to correct it, it's just nice when things line up sometimes :P 
You can also use `@desc` instead of `@description`. Again, just giving you information, not telling you what you have to do <sub>(in this case)</sub>